### PR TITLE
Made a meme command with buttons - revamped unmute

### PR DESCRIPTION
**Pull request by RopraMMC**

### CODE REVIEW

File: src/cogs/mod.py [hunk 1]
```diff
@@ -216,14 +216,18 @@ async def timeout(self, ctx, member: disnake.Member, time: str, *, reason=None):
...
-    async def unmute(self, ctx, member: disnake.Member, *, reason=None):
+    async def unmute(self, ctx: Context, member: disnake.Member, *, reason=None):
         """unmutes a member (or removes timeout) from a guild"""
         await member.timeout(until=None, reason=reason)
-        await ctx.send(
-            embed=disnake.Embed(
-                title="unmuted", description=f"{member.mention} was unmuted."
-            )
-        )
+        embed=disnake.Embed(
```
Comment (Yat-o) [inline src/cogs/mod.py:222]: What was the point in building the embed outside the send() method

Comment (RopraMMC) [reply src/cogs/mod.py:222]: thanks

Comment (RopraMMC) [reply src/cogs/mod.py:222]: because I like variables

Comment (sifte) [reply src/cogs/mod.py:220]: Don't make useless variables

**Review by sifte — DISMISSED** [review]

**Review by Pandabweer — APPROVED** [review]

**Comment by RopraMMC** [comment]
2ND REVIEWER: DONT MERGE YET I NEED TO STILL DO SOME STUFF

**Comment by RopraMMC** [comment]
GIVE ME 20HRS

**Comment by Dekriel** [comment]
wow

**Comment by RopraMMC** [comment]
ok yeah I'm done

**Review by sifte — APPROVED** [review]

**Pull request merged**